depthai_demo.py

```diff
@@ -562,6 +568,9 @@ def create_system_logger(self):
 
     if conf.useCamera:
         pv.create_queues(device)
+    elif conf.args.sync:
+        conf.args.show = ["host"]
```

Comment:
I would move this to adjustPreviewToOptions - https://github.com/luxonis/depthai/blob/1f2244c75a660272bdef15f1865a4fda8b3fcd6a/depthai_helpers/config_manager.py#L184